### DISCUSSION TITLE
Speech Hints

### DISCUSSION THREAD

**Discussion by ayushch4** [deepgram/community#825] [General help] [answered]
Hi Team,

I am trying to figure out what's the best way to send speech hints when we transcribe. Invariably, company names gets incorrectly transcribed along with product names. I tested keywords with company_name:1, not sure what's the recommended approach?

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by davidvonthenen** [comment]
hi @ayushch4 

That is pretty much correct. Keywords is going to be your best bet. You can find more about keywords here:
https://developers.deepgram.com/docs/keywords

If you find that it is getting translated for some odd reason to something very consistent, you can use find/replace:
https://developers.deepgram.com/docs/find-and-replace

**Reply by ayushch4** [reply]
@dvonthenen Here is a challenge we are running into, my company name is Vidoso. Its transcribing video to Vidoso as well and on the flip side, without keywords, it transcribes our company Vodoso if we don't use keywords. Any suggestions on best way to handle this?

**Reply by davidvonthenen** [reply]
So, if I understand correctly, you should do `No Keywords` but `Find/Replace` from `Vodoso` -> to `Vidoso`.

**Reply by ayushch4** [reply]
@dvonthenen That would work if our company name was transcribed Vodoso always, its always a new name that the STT transcribes. I have seen Podoso and Vodosa.

Essentially, what I am asking for is the keyword only boosts the company name, but not every similar sounding/phoentically similar word. The side effect is any time some one says Video, it will be transcribed to Vidoso. If we don't use keywords, we will get some 5 variants of the company name and find/replace won't work, if we don't know what STT will transcribe

**Reply by davidvonthenen** [reply]
Hi @ayushch4 

I think there are improvements in Keywords coming soon. I don't have the details on that though.
Another option is a custom model, but that definitely requires a contract.
I will see about how this is handled in other ways since this is my first time encountering this problem.

**Reply by SandraRodgers** [reply, marked as answer]
@ayushch4 You could also try boosting the keyword with an intensifier that is smaller than 1. The intensifier value can be a decimal.

This isn't a sure fix, but more a trial and error fix to see if it gets you closer. I would recommend using the slight intensifier until you no longer see words like Video getting transcribed inaccurately. Combine that with the find and replace that dvonthenen suggested . Not a perfect solution, unfortunately, but it could help.
